openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -204,7 +204,7 @@ This is an example of a Wallet Instance Attestation:
    * MUST support protocol extensions for SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
    * As a way to invoke the Wallet, at least a custom URL scheme `haip://` MUST be supported. Implementations MAY support other ways to invoke the wallets as agreed by trust frameworks/ecosystems/jurisdictions, not limited to using other custom URL schemes.
    * Response type MUST be `vp_token`.
-   * Response mode MUST be `direct_post` with `redirect_uri` as defined in Section 6.2 of [@!OIDF.OID4VP].
+   * Response mode MUST be `direct_post`. The Verifier MUST return `redirect_uri` in response to the HTTP POST request from the Wallet, where the Wallet redirects the User to, as defined in Section 6.2 of [@!OIDF.OID4VP]. Implementation considerations for the response mode `direct_post` are given in Section 11.5 of [@!OIDF.OID4VP].
```

Comment:
Are we sure we want the redirect_uri in cross device scenarios, too? Otherwise, this text is to generic.

Reply:
that's true - question is whether this means HAIP must be narrowed to same device or whether there are situations where this could be otherwise detected (note: this is a problem with all oob protocols)
@danielfett what do you think?

Reply:
this probably needs to be clarified in 4VP spec itself. there is an issue https://github.com/openid/OpenID4VP/issues/25
but if we can go ahead and limit it in HAIP, I would not againt it, but probably out of scope for this PR.